packages/cli/src/analytics/analytics.ts

```diff
@@ -1,11 +1,44 @@
+import { LightdashUser } from '@lightdash/common';
 import fetch from 'node-fetch';
 import { v4 as uuidv4 } from 'uuid';
+import { Config, getConfig, setAnonymousUuid } from '../config';
+import { lightdashApi } from '../handlers/dbt/apiClient';
 
 const { version: VERSION } = require('../../package.json');
 
+const identifyUser = async (): Promise<Config['user']> => {
+    const config = await getConfig();
+    let anonymousUuid = config.user?.anonymousUuid;
+    if (anonymousUuid === undefined) {
+        anonymousUuid = uuidv4();
+        await setAnonymousUuid(anonymousUuid);
```

Comment:
should we generate the anonymous uuid inside `getConfig`

Reply:
Yes for sure! I actually moved it but that works too

Reply:
Done!